Tasks/HelmDeployV0/Tests/L0.ts

```diff
@@ -1,16 +1,194 @@
 import fs = require('fs');
 import assert = require('assert');
 import path = require('path');
+import * as ttm from 'azure-pipelines-task-lib/mock-test';
+import * as shared from './TestShared';
+import * as tl from 'azure-pipelines-task-lib';
 
 describe('HelmDeployV0 Suite', function () {
-    before(() => {
+    this.timeout(30000);
+
+    before((done) => {
+        process.env[shared.TestEnvVars.operatingSystem] = tl.osType().match(/^Win/) ? shared.OperatingSystems.Windows : shared.OperatingSystems.Other;
+        done();
+    });
+
+    beforeEach(() => {
+        delete process.env[shared.TestEnvVars.namespace];
+        delete process.env[shared.TestEnvVars.valueFile];
+        delete process.env[shared.TestEnvVars.overrideValues];
+        delete process.env[shared.TestEnvVars.updatedependency];
+        delete process.env[shared.isHelmV3];
+        delete process.env[shared.TestEnvVars.releaseName];
+        delete process.env[shared.TestEnvVars.waitForExecution];
+        delete process.env[shared.TestEnvVars.arguments];
+        delete process.env[shared.TestEnvVars.chartName];
+        delete process.env[shared.TestEnvVars.chartPath];
+        delete process.env[shared.TestEnvVars.connectionType];
+        delete process.env[shared.TestEnvVars.command];
+        delete process.env[shared.TestEnvVars.chartType];
+    });
+
+    after((done) => {
+        done();
+    });
+
+    it('Run successfully with Helm install (version 3) with chart name', function(done: MochaDone) {        
+        const tp = path.join(__dirname, 'TestSetup.js');
+        const tr: ttm.MockTestRunner = new ttm.MockTestRunner(tp);
+        process.env[shared.TestEnvVars.connectionType] = shared.ConnectionTypes.KubernetesServiceConnection;
+        process.env[shared.TestEnvVars.command] = shared.Commands.install;
+        process.env[shared.TestEnvVars.chartType] = shared.ChartTypes.Name;
+        process.env[shared.TestEnvVars.chartName] = shared.testChartName;
+        process.env[shared.TestEnvVars.releaseName] = shared.testReleaseName;
+        process.env[shared.TestEnvVars.failOnStderr] = 'ture';
+        process.env[shared.isHelmV3] = 'true';
+
+        tr.run();
+        assert(tr.stdout.indexOf("v3") != -1, 'Helm version 3 installed');
```

Comment:
inconsistent quotes across the file

Reply:
Will update to double quotes everywhere. However, some strings need to use `  as they are using variables.